src/components/__tests__/root.spec.js

```diff
@@ -1,5 +1,8 @@
 import defaultValues from '../../__tests__/default-button-props';
-import { render } from '../root';
+import { render, teardown } from '../root';
+
+// Empty function should be test when filled
+teardown();
```

Comment:
Probably taking it too far here, but would be great if we can have all files except index.js fully covered

Reply:
So you are just running the function to get the coverage? I actually think we should wait to write an actual test

Reply:
I don't think we will implement the teardown function. I'm actually thinking of removing this completely

Reply:
@haxxmaxx So removed the teardown function. Pretty sure we are not going to implement anything here

Reply:
Great!